pilot/lib/pilot/build_manager.rb

```diff
@@ -273,8 +273,14 @@ def self.truncate_changelog(changelog)
       changelog
     end
 
+    def self.emoji_regex
+      # EmojiRegex::RGIEmoji is now preferred over EmojiRegex::Regex which is deprecated as of 3.2.0
+      # https://github.com/ticky/ruby-emoji-regex/releases/tag/v3.2.0
+      return defined?(EmojiRegex::RGIEmoji) ? EmojiRegex::RGIEmoji : EmojiRegex::Regex
+    end
+
     def self.strip_emoji(changelog)
-      if changelog && changelog =~ EmojiRegex::Regex
+      if changelog && changelog =~ emoji_regex
         changelog.gsub!(EmojiRegex::Regex, "")
```

Comment:
What about this one? Shouldn't we...

```suggestion
        changelog.gsub!(emoji_regex, "")
```

Reply:
lol, you are right